Sitka-Sound-Science-Center/sitka-landslide
# Update static copy and tiles

## Overview

Update static copy across application.

Resolves #12

### Demo

<img width="384" alt="Screen Shot 2022-05-04 at 12 44 14 PM" src="https://user-images.githubusercontent.com/1809908/166730851-d0f36e59-1080-498d-915b-e233524616e3.png">

## Testing Instructions

 * Scan each page and make sure nothing looks broken
 * I don't think anyone needs to go through line by line to make sure it matches—the client can let us know if there are any detailed issues


## Checklist
